src/api/wix/WixToolset.Data/Symbols/WixBundleVariableSymbol.cs

```diff
@@ -35,6 +35,7 @@ public enum WixBundleVariableAttributes
         None = 0x0,
         Hidden = 0x1,
         Persisted = 0x2,
+        BuiltIn = 0x4,
```

Comment:
Can variables not on the static builtin-list be declared as "builtin"? I can't think of a case where that makes sense, so I wonder if I'm missing a scenario.